EVGEN/AliDecayer.h

```diff
@@ -20,7 +20,8 @@ typedef enum
     kAll, kNoDecayHeavy, kHardMuons, kBJpsi,  kBJpsiUndecayed,
     kWToMuon,kWToCharm, kWToCharmToMuon, kZDiMuon, kZDiElectron, kNeutralPion, kAllMuonic,
     kChiToJpsiGammaToMuonMuon, kChiToJpsiGammaToElectronElectron, kNoDecayBeauty, kPsiPrimeJpsiDiElectron,
-  kElectronEM, kGammaEM, kDiElectronEM, kBeautyUpgrade,kHadronicDWithV0,kHadronicDWithout4BodiesWithV0,kAllEM,
+  kElectronEM, kGammaEM, kDiElectronEM, kBeautyUpgrade,kHFYellowReport,
```

Comment:
Hi, trivial question maybe, but here kHFYellowReport is added in between of the list of elements of Decay_t . Are we sure that at other points in the code, Decay_t isn't compared to integer values ?

Reply:
This is a very good point. It must be added at the end of the enum definition in any case. @btrzecia please update the PR as requested by @rbertens.

Reply:
Hi, indeed that was not a safe solution, thanks for pointing this out.